src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -325,14 +334,22 @@ export default class AddStaffNotationDialog extends Vue {
         }
       }
     } else if (this.isCourtOrder) {
-      filing[FilingTypes.COURT_ORDER] = {
-        effectOfOrder,
-        fileNumber,
-        orderDetails: this.notation,
-        fileKey: this.fileKey,
-        fileName: this.file.name,
-        fileLastModified: this.file.lastModified,
-        fileSize: this.file.size
+      if (this.file) {
+        filing[FilingTypes.COURT_ORDER] = {
+          effectOfOrder,
+          fileNumber,
+          orderDetails: this.notation,
+          fileKey: this.fileKey,
+          fileName: this.file.name,
+          fileLastModified: this.file.lastModified,
+          fileSize: this.file.size
+        }
+      } else {
+        filing[FilingTypes.COURT_ORDER] = {
+          effectOfOrder,
+          fileNumber,
+          orderDetails: this.notation
+        }
```

Comment:
Thanks for the fixes in here. Is there a bug ticket for these that you can take credit for?

(If not, please identify these changes in a Test Notes comment in your ticket so that UX Team and QA know to verify this.)

Reply:
Added this in the ticket details. Will add Test notes to make sure it is covered in testing.

Reply:
I see those comments in this PR's description but not in ticket 14602. I believe UX and QA don't look at the PRs much so please add to the ticket as well.

Reply:
Already there :)
![image](https://user-images.githubusercontent.com/116035339/213269345-d57c1f78-3c29-4543-98c9-18960e7263a8.png)